content/Mobile_Dev/index.adoc

```diff
@@ -0,0 +1,64 @@
+== Multi-Factor Authentication for Mobile Devices
+Protecting user accounts does not stop at desktops and personal computers. Consumers have continually shifted to mobile devices as their primary mechanism for connecting to the web. Many enterprises have already adopted, or have begun to adopt, mobile device management solutions to allow their people to connect to enterprise apps through company or personal devices. Your adaptive multi-factor authentication strategy will be at a huge disadvantage if you do not consider the experience for users who are connecting with their mobile devices. Through this series we are going to explore a variety of different concepts on enabling MFA on mobile devices. By the end of this series you should have the tools, and knowledge on how to select a MFA approach, and how to implement it into your own solution. 
+
+=== Why do mobile devices need specific consideration in authentication strategies?
+As noted above, there has been continued growth in adoption of mobile devices in both the consumer and enterprise space. In this section we are going to outline a few different factors on why this is the right time to take advantage of the needs and requirements of users in different spaces, and merge them with functionality built into mobile devices.
```

Comment:
1. The phrase 'as noted above' is not necessary here. Consider removing it.

Reply:
Changed in latest commit